.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+Suggest features in the Mattermost feature idea forum so it can be discussed, upvoted and considered for a [help wanted ticket](https://docs.mattermost.com/process/help-wanted.html).
```

Comment:
Same here. For plugins, suggestions should to into GitHub. Should this template be limited to Mobile and Server?